lua/astrocommunity/pack/python/init.lua

```diff
@@ -88,8 +91,15 @@ return {
         },
       },
     },
-    opts = {},
-    cmd = "VenvSelect",
+    opts = {
+      -- options = {
```

Comment:
No commented out code should be in the plugin. If you wish, then can indicate in the readme some options or configuration.